src/main/resources/application-local.yml

```diff
@@ -21,6 +21,7 @@ server:
 
 court-case-service:
   base-url: http://localhost:8082/
+  case-by-hearing-id-court-case-id-url-template: /hearing/%s/case/%s
```

Comment:
I know this pattern is used in this application elsewhere, but I must confess I find it strange. This template will not change by environment. There is no case for changing it without deploying the code. Why not simply make it a constant in `CourtCaseServiceClient`? It will save developers having to look it up

Reply:
Yes, good point. I have inlined it along with the others referenced in the client service.